packages/harpguru-core/src/hooks/use-octave-column-groups/use-octave-column-groups.test.ts

```diff
@@ -1,6 +1,6 @@
 import { useGlobal } from 'reactn'
```

Comment:
It would definitely be worth considering keeping this function clear of the override, and having  it always return the octave column groups. It could then be up to the caller to flatten it if it were relevant.

Reply:
`useOctaveColumnGroups` & `useFragmentColumnGroups`

Reply:
Done. The size scheme is no longer aware of the column groups so there is no need for 2 hooks.